aemsites/hdfc-bank
# Nrenro develop2 (#845)

* create new branch

* reverted constant.js changes and added path mapping to functions.js

* NRE-NRO CSS

* add constant file for endpoint changes

* changes done for landing page

* ui-fixes

* revert one change

* Update fstab.yaml

* Update fstab.yaml

* Update fstab.yaml

* ui-changes

* ui changes

first and second screen ui changes

* remove local link

* updated consent popup changes

* updated customFocus function

* select-account screen

* Minor fixes

* add checkbox arrow

* css update responsive first and second screen

* add prefills of customer personal, fatca and other details

* UI update

select-account and confirm-details page

* testurl updated

* updated url for testing

* create error page

Add CSS for the wrong mobile number error page and the error page that appears after three incorrect entries. Additionally, implement a new function to add a class to the body based on the API response.

* move css inside .nre class parent

move css inside .nre class parent

* ISD and Non-Isd validation

* IDCom Redirection functions

* responsive screen

error screen and wizard screen responsive

* Update fstab.yaml

* invoke journey for select account and add isd code with mobile number

* splited dob with -

* refactor css

* fix mob input

* fix small ui

* Update functions.js

* fixed UI bugs and format page

working with accordian and fixed UI bugs and format page

* Update nre-nroFunctions.js

* Confirm Screen analytics impl commit

* Code cleanup commit

* Finacial details

* updated finacial and nomineedetails functions

* thank-you-page-design

* Update nre-nroFunctions.js

* merge conflict fixed

* copy button and button hover update

* thanks-you page textarea

* Update constant.js

* The thank you page is now responsive on iPad

The thank you page is now responsive on iPad, and the CSS overwrite issue with HDFC Bank has been fixed.

* Thank you page is now responsive in mobile view

* fix small testing bug fixes

* fix confirm-details screen click to accordian hide everything fixed.

* last commit revert

* add functionality for multiple account sccren and masking loigic

* IdCom redirection code cleanup commit

* update radio button formultiple account

* small responsive udpate

* add multiple account css

* SetupBankUse section changes commit

* Reverting deleted files commit

* Reverting Help.svg deletion commit

* test bug 40-41

* Review comments incorporated commit

* Review comments incorporated commit

* Lint issues fix commit

* Minor change

* ui-udpate and bug fixes

* thanks page update

* date of birth

* CustID Mapping fix for Idcom

* Minor cleanup commit

* ininvokeJourneyUpdate fixes commit

* crm leadid api and select account from multi account

* remove hardcoded value from crm leadid api

* Minor changes

* crmProduct details

* crmLeadIdDetail updtaed changes

* Minor changes

* Minor changes

* IDCom return journey update commit

* Lint issues fix

* Minor changes

* prefilld details on idecom redirect

* qa-observation-ui

* prefill financial details

* accordian functionality

* match mobile field as per figma

* current account variant it is not visible properly-fixed

* Post idCom to Thank You Page commit

* accordian-fixed

* update the code for  Nre-Nro accounttype varients and nominee details section

* updated the prefillCustomerDetail function

* Minor changes

* updtae journeydropoff and journeydropoffupdate with ISDcode

* updated countryCode

* accordion-fixed

* field-nomineedetails condition css update

* updated changes for multiple acccount

* space between customerID and disabled toggle button

* lg code css udpate as input not prefield

* removed spaces for permanent address

* Minor changes

* Thank you page changes in progress

* accordian hiddne fields fixed

* responsive-ui-fixed

* Till before Thank You page changes commit

* Country code temp fix commit

* Styles.css revert

* Styles.css issue reverted

* error handling changes and change in masking logic of customerid and account number

* change in eslint file

* internal-demo-points

* demo points update

* show success popup in header

* feat:make createFormInstance wait for async call (#829)



* Update nre-nroFunctions.js

* Update nre-nroFunctions.js

* stage fd fixes (#839)

* index changes

---------

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--{repo}--{owner}.hlx.live/
- After: https://<branch>--{repo}--{owner}.hlx.live/
